bin/cml/comment/create.js

```diff
@@ -55,5 +55,12 @@ exports.options = kebabcaseKeys({
     type: 'boolean',
     description:
       'Avoid watermark; CML needs a watermark to be able to distinguish CML comments from others'
+  },
+  commentLabel: {
+    type: 'string',
+    description:
+      'Label to add to watermarks for more selective comment updates',
+    default: '',
```

Comment:
does it make sense to do this by default?

```suggestion
    default: '{workflow}',
```

Reply:
That would alter the current behavior.

Reply:
yes, it would - but is that a gewd thing?

Reply:
I've changed the url of the image used in the watermark - it's now served from cml.dev